src/handler/mod.rs

```diff
@@ -140,6 +140,17 @@ pub enum ConnectionDirection {
     Outgoing,
 }
 
+/// An instruction from the handler to the service how to update the connection direction.
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub enum ConnectionDirectionInstruction {
```

Comment:
What would you think or `ConnectionUpdate` simply because it's shorter? `ConnectionChange` in another option

Reply:
Looks good, thank you. 💡